generate.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -o pipefail
+
+MYDIR=$(pwd)
+SCAFFOLD_VERSION_OR_BRANCH="generator-changes"
+SCAFFOLD_URL="https://github.com/mobify/progressive-web-scaffold/archive/$SCAFFOLD_VERSION_OR_BRANCH.zip"
+
+# Prompt license and do not proceed unless user has accepted
+read -p"--> We have a license you must read and agree to. Read license? (y/n) " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
+    exit 1
+fi
+
+curl -s -O -L https://raw.githubusercontent.com/mobify/progressive-web-scaffold/develop/LICENSE
+trap 'rm -f LICENSE' EXIT
+less LICENSE
+
+read -p"--> I have read, understand, and accept the terms and conditions stated in the license above. (y/n) " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
+    exit 1
+fi
+
+# Get the project slug from the user
+read -p'--> What is your project slug? (should match slug on Mobify Cloud) ' project_slug
+
+# Get the project url from the user
+read -p'--> What is the project url? ' project_url
+
+# $project_slug must not contain special characters.
+project_slug=$(echo "$project_slug" | tr -dc '[:alnum:]\n\r' | tr '[:upper:]' '[:lower:]' | tr -d ' ')
+
+read -p"--> Continue with the project name '$project_slug'? (y/n) " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
+    exit 1
+fi
+
+# Prepare new project directory
+project_dir="$MYDIR/$project_slug"
+echo "Setting up new project in $project_dir"
+mkdir "$project_dir"
+cd "$project_dir" || exit
+
+printf "\nDownloading Progressive Web project scaffold\n"
+WORKING_DIR=$(mktemp -d /tmp/progressive-web-scaffold.XXXXX)
+trap 'rm -rf "$WORKING_DIR"' EXIT
+curl --progress-bar -L "$SCAFFOLD_URL" -o "$WORKING_DIR/progressive-web-scaffold-$SCAFFOLD_VERSION_OR_BRANCH.zip"
+cd "$WORKING_DIR" || exit
+unzip -q "$WORKING_DIR/progressive-web-scaffold-$SCAFFOLD_VERSION_OR_BRANCH.zip"
```

Comment:
Sorry not sure what this is suggesting, are you suggesting `unzip` without the `-p`?